examples/space_invaders/minimal_space_invaders.py

```diff
@@ -1,4 +1,5 @@
 import torch
```

Comment:
Can you revert these changes? This "minimal" script is used as an example in the BindsNET paper.